docs/tutorials/educators/intro.ipynb

```diff
@@ -2,14 +2,14 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    print("Before optimizing:\n{}\n".format(circuit))
</pre><p>slight pref for f-strings these days.  Probably should at least be consistent across the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='160'/>